packages/signature-controller/CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [26.0.0]
+
+### Added
+
+- **BREAKING:** Add peer dependency on `^26.0.0` of `@metamask/accounts-controller`([#5470](https://github.com/MetaMask/core/pull/5470))
+- Add EIP-7702 signature validations ([#5470](https://github.com/MetaMask/core/pull/5470))
+  - Throw if external and `verifyingContract` matches any internal account.
+  - Throw if external and `primaryType` is `Delegation` and `delegator` matches any internal EOA account.
+
+### Changed
+
+- Bump `@metamask/accounts-controller` peer dependency to `^26.1.0` ([#5481](https://github.com/MetaMask/core/pull/5481))
```

Comment:
Should we merge the `accounts-controller` entries? The initial change to introduce 26.0.0 wasn't ever public and then we bumped the dependency to 26.1.0 so we can just say that the peer dependency is 26.1.0.

``` suggestion
- **BREAKING:** Add peer dependency on `^26.1.0` of `@metamask/accounts-controller` ([#5470](https://github.com/MetaMask/core/pull/5470), [#5481](https://github.com/MetaMask/core/pull/5481))
- Add EIP-7702 signature validations ([#5470](https://github.com/MetaMask/core/pull/5470))
  - Throw if external and `verifyingContract` matches any internal account.
  - Throw if external and `primaryType` is `Delegation` and `delegator` matches any internal EOA account.